trusted-types/trusted-types-svg-script.html

```diff
@@ -25,43 +25,12 @@
 
     promise_test(async t => {
       const input = "'modified via innerHTML';"
-      let violation = await trusted_type_violation_for(TypeError, _ =>
-        document.getElementById("script").innerHTML = policy.createHTML(input)
-      );
-      assert_true(violation.originalPolicy.includes("require-trusted-types-for 'script'"));
-      assert_equals(violation.sample, `Element innerHTML|${clipSampleIfNeeded(input)}`);
+      document.getElementById("script").innerHTML = policy.createHTML(input);
+      return Promise.resolve();
```

Comment:
This does not contain any assertion, I guess we should wrap into a `no_trusted_type_violation_for` to at least keep testing the (absence of) violation report.